src/common/verify/isValidAccessGrant.ts

```diff
@@ -28,8 +28,10 @@ import {
   getIssuer,
   getVerifiableCredentialApiConfiguration,
 } from "@inrupt/solid-client-vc";
+import { isUrl } from "@inrupt/solid-client-vc/dist/common/common";
```

Comment:
```suggestion
import { isUrl } from "@inrupt/solid-client-vc/common";
```
This should be aligned with the `exports` in https://github.com/inrupt/solid-client-vc-js/blob/c18412db41f03aeab1024fef0b63b28bf7ba0bad/package.json#L34.

Reply:
Ah interesting, it doesn't resolve the issue.